internal/characters/traveler/common/hydro/travelerhydro.go

```diff
@@ -0,0 +1,51 @@
+package hydro
+
+import (
+	"github.com/genshinsim/gcsim/internal/common"
+	tmpl "github.com/genshinsim/gcsim/internal/template/character"
+	"github.com/genshinsim/gcsim/pkg/core"
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/player/character"
+	"github.com/genshinsim/gcsim/pkg/core/player/character/profile"
+)
+
+type char struct {
+	*tmpl.Character
+	droplets []*common.SourcewaterDroplet
```

Comment:
Check if HMC can pick up Neuvilette's Sourcewater droplets? Because the current implementation will prevent HMC from picking up Neuvilette's droplets

Reply:
Yes, HMC can pick up Neuvillette's Sourcewater Droplets.

Reply:
out of scope for the pr